docs/src/pages/components/cdk-components/resultsetTable/api.jsx

```diff
@@ -48,6 +48,13 @@ const resultsetTablePropsTable = () => {
           </ul>
         </td>
       </tr>
+      <tr>
+        <td>showGoToPage: boolean</td>
+        <td>
+          <code>true</code>
+        </td>
+        <td>Show page navigation select.</td>
```

Comment:
I think we should extend the explanation. Something like: "If true, a Select for navigation between pages will be displayed".

Reply:
Done.